package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "terra-toolkit",
-  "version": "3.6.0",
+  "version": "3.5.0",
```

Comment:
Should this be `4.0.0`?

Reply:
Not yet (until we release it)

Reply:
Downgrading seemed odd to me

Reply:
This PR has been out for quite a while. The downgrade happened because we've had a release since this PR was put out. I will update.